bazel/repositories.bzl

```diff
@@ -11,7 +11,9 @@ def nighthawk_dependencies():
         name = "envoy",
         sha256 = ENVOY_SHA,
         strip_prefix = "envoy-%s" % ENVOY_COMMIT,
+        # // clang-format off: Envoy's format check: Only repository_locations.bzl may contains URL references
```

Comment:
This seems to be the first instance where us just using envoy's .bazelrc isn't great for us.  No action required here on this PR, but we should keep an eye on it.